www_src/lib/snackbar-mixin.js

```diff
@@ -0,0 +1,56 @@
+var React = require("react/addons");
+var Snackbar = require('../components/snackbar/snackbar.jsx');
+
+module.exports = {
+  /**
```

Comment:
this still needs a `componentDidMount` function that sets up dispatcher listening for "webmaker:error" or the like, which can then trigger the `recordError` function, for elements that do not have direct reference access to the Snackbar element.